docs/modules/clientutils.rst

```diff
@@ -135,7 +135,26 @@ Retrieves a single DOM element by a :ref:`selector expression <selectors>`::
 
     casper.run(function() {
         this.echo(href).exit();
+    });    
+    
+``forceTarget()``
+-------------------------------------------------------------------------------
+
+**Signature:** ``forceTarget(String selector, String target)``
+
+Force a browsing to use another target instead the one provided. Very useful to limitate window number and reduce memory::
+
```

Comment:
"Force the engine to use another target instead of the one provided. Very useful to limit the number of open windows and reduce memory consumption."